src/aat/resources/features/F-054/F-054.feature

```diff
@@ -0,0 +1,129 @@
+
+  @S-562 @Ignore # Response code mismatch, expected: 400, actual: 200 /RDM-7066
+  Scenario: must return negative response HTTP-400 when request contains a non-existing jurisdiction ID
+    Given a successful call [to create a token for case creation as a citizen] as in [Citizen_Token_Creation_Data_For_Case_Creation]
+    And   another successful call [to create a full case as a citizen] as in [Citizen_Full_Case_Creation_Data]
+    And   a user with [an active profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains a non-existing jurisdiction ID]
+    And   it is submitted to call the [get case for citizen] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-400]
+    And   the response has all other details as expected
+
+  @S-563 @Ignore # Response code mismatch, expected: 400, actual: 200 /RDM-7066
+  Scenario: must return negative response HTTP-400 when request contains a non-existing case type ID
+    Given a successful call [to create a token for case creation as a citizen] as in [Citizen_Token_Creation_Data_For_Case_Creation]
+    And   another successful call [to create a full case as a citizen] as in [Citizen_Full_Case_Creation_Data]
+    And   a user with [an active profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains a non-existing case type ID]
+    And   it is submitted to call the [get case for citizen] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-400]
+    And   the response has all other details as expected
+
+  @S-564
+  Scenario: must return negative response HTTP-400 when request contains a non-existing case reference ID
+    And   a user with [an active profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains a non-existing case reference ID]
+    And   it is submitted to call the [get case for citizen] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-400]
+    And   the response has all other details as expected
+
+  @S-565
+  Scenario: must return negative response HTTP-403 when request contains a non-existing user ID
+    Given a successful call [to create a token for case creation as a citizen] as in [Citizen_Token_Creation_Data_For_Case_Creation]
+    And   another successful call [to create a full case as a citizen] as in [Citizen_Full_Case_Creation_Data]
+    And   a user with [an active profile in CCD]
+    When  a request is prepared with appropriate values
+    And   the request [contains a non-existing user ID]
+    And   it is submitted to call the [get case for citizen] operation of [CCD Data Store]
+    Then  a negative response is received
+    And   the response [code is HTTP-403]
+    And   the response has all other details as expected
```

Comment:
proceeding lines should contain a wording to relate to the initial case created.

Reply:
done